PowerEditor/installer/nativeLang/bulgarian.xml

```diff
@@ -317,7 +317,7 @@
 				<!-- Меню "Изглед" край -->
 				<!-- Меню "Кодировки" начало -->
 					<Item id="45004" name="Кодиране в ANSI"/>
-					<Item id="45008" name="Кодиране в UTF-8"/>
+					<Item id="45008" name="Кодиране в UTF-8 без BOM"/>
```

Comment:
I remove this `без BOM` because in the status bar it is only `UTF-8`

`без BOM` е умишлено премахнато, тъй като в лентата на състоянието се показва само `UTF-8`. Идеята ми е да бъде еднакво навсякъде. Ако ще се променя на `UTF-8 без BOM` трябва да се редактират: `<Item id="45010" name="Преобразуване в UTF-8"/>` и `<Item id="6407" name="UTF-8"/>`. Мисля, че се подразбира, че щом едното е с `BOM`, то другото е `без BOM`

Reply:
Да, в оригинала е така. Предлагам ти да изчакаме #4040  #4046 за да не се налага после да връщаме :)

Reply:
Ще съм благодарен и улеснен, ако в бъдеще се използва `Форум за дискусии можете да намерите на: https://www.kaldata.com/forums/topic/232576-превод-на-notepad/`, а тук да се публикуват само изчистени вече неща.

Reply:
Let's remain UTF-8 without BOM part